src/components/motion/MotionController.h

```diff
@@ -32,15 +43,19 @@ namespace Pinetime {
         return zHistory[0];
       }
 
-      uint32_t NbSteps() const {
+      step_t NbSteps() const {
```

Comment:
I think it'd be better if NbSteps took a member of Days as an argument  (as you suggest actually)
The step counter history method is still needed though if you plan on exposing this over BLE

Reply:
Done

I've set `Today` as default parameter.

Should I revert most of the changes to `Steps.cpp`, where I'm storing a const reference to `nbSteps`?

Reply:
I would personally as I think calling NbSteps() is a bit clearer than using the buffer reference